.github/workflows/ci.yml

```diff
@@ -40,15 +40,19 @@ jobs:
           python-version: ${{ matrix.python.action }}
           architecture: ${{ matrix.arch.action }}
 
-      - name: boots
+      - name: Install poetry ${{ matrix.poetry-version }}
         run: |
-          python boots.py ensure --group build
+          python -m ensurepip
+          python -m pip install --upgrade pip
+          python -m pip install poetry
 
-      - name: Package executable
+      - name: poetry install
         run: |
-          # help versioneer and pyinstaller get along
-          venv/Scripts/pip install .
-          venv/Scripts/pyinstaller pyinstaller.spec
+          poetry install
```

Comment:
Maybe move this under Install poetry? Maybe the wording is just weird. I know it's installing the poetry environment and not installing poetry.